app/docker/views/networks/edit/NetworkDetailsTable.tsx

```diff
@@ -0,0 +1,111 @@
+import { Fragment } from 'react';
+
+import { Widget, WidgetBody, WidgetTitle } from '@/portainer/components/widget';
+import { Button } from '@/portainer/components/Button';
+import { Authorized } from '@/portainer/hooks/useUser';
+
+import { NetworkRowContent, IPConfigs } from '../types';
+
+interface Props {
+  networkRowContent: NetworkRowContent | undefined;
+  allowRemove: boolean;
+  IPV4Configs: IPConfigs;
+  IPV6Configs: IPConfigs;
+  onRemoveNetworkClicked: () => void;
+}
+
+export function NetworkDetailsTable({
```

Comment:
since this table shows the details for the network, and in the future, we might want more info, I think it's more appropriate to pass the network object instead and do data manipulation in this component.

providing "NetworkRowContent" means the component user (in this case, the root view), knows too much about the implementation of this component. it also adds complexity to the root view.

Reply:
That makes sense to me - the latest commit passes in the network object to the NetworkDetailsTable, and not the NetworkRowContent

Reply:
:+1:

Reply:
I suggest do the same with the other data that is parsed from the network object (and isn't needed in the root view)